frontend/src/features/rateLater/autoSuggestions.ts

```diff
@@ -0,0 +1,33 @@
+export const AUTO_SUGGESTIONS: { [key: string]: string[] } = {};
+
+export const clearAutoSuggestions = () => {
+  Object.keys(AUTO_SUGGESTIONS).forEach((key) => (AUTO_SUGGESTIONS[key] = []));
+};
+
+export const isAutoSuggestionsEmpty = (poll: string) =>
+  AUTO_SUGGESTIONS[poll] == null || AUTO_SUGGESTIONS[poll].length === 0;
+
+export const fillAutoSuggestions = (
+  poll: string,
+  uids: string[],
+  exclude?: string[]
+) => {
+  AUTO_SUGGESTIONS[poll] = uids.filter((uid) => !exclude?.includes(uid));
```

Comment:
This operation and a few others in these methods are unnecessarily of quadratic complexity.
Casting exclude to a `Set` and calling `excludeSet.has` should be faster. Especially if we crank it up in the future to more than 100 uids.

(see https://www.tech-hour.com/javascript-performance-and-optimization)

Reply:
I see after this comment that it is only used with the exclude list of length 2, so no impact for the current usage. But I'd still recommend the very easy optimization to protect us from future dev using these arguments differently

Reply:
Good, fixed by https://github.com/tournesol-app/tournesol/pull/1835/commits/2e5afe38c70a901d4e45712a20be2402148d4f6e